html2docx/html2docx.py

```diff
@@ -9,8 +10,19 @@
 from tinycss2 import parse_declaration_list
 from tinycss2.ast import IdentToken
 
+from .image import get_image_size
+
 WHITESPACE_RE = re.compile(r"\s+")
 
+DEFAULT_DPI = 72
```

Comment:
Where is this used?

Actually docx has a function for converting values that is used like this:
```
from docx.shared import Pt
points = Pt(pixels)
```

Reply:
It’s not used anymore, it’s a leftover from an earlier version of the code. Thanks for pointing that out, I removed it.